docs/Upgrade.md

```diff
@@ -1380,7 +1380,7 @@ So the `RecordMap` type is no longer necessary and was removed. TypeScript compi
 
 ```diff
 -import { useGetList, RecordMap } from 'react-admin';
-+import { useGetList, Record } from 'react-admin';
++import { useGetList, RaRecord as Record } from 'react-admin';
```

Comment:
We should avoid renaming it imo

Reply:
#7563 is correct. I'm closing this one in favor of that PR